int32 drill_current_direction
int32 drill_current_speed

bool direction
